src/CabanaController.hpp

```diff
@@ -81,10 +80,11 @@ class CabanaController {
   using TypeTuple = std::tuple<Ts...>;
   using DeviceType = Kokkos::Device<ExecutionSpace, MemorySpace>;
   using DataTypes = Cabana::MemberTypes<Ts...>;
-  
+
 public:
   typedef ExecutionSpace exe;
-  static const int MAX_RANK = 4; // Including num_tuples -> so 3 additional extents
+  static const int MAX_RANK =
+      4; // Including num_tuples -> so 3 additional extents
```

Comment:
```suggestion
  // Including num_tuples -> so 3 additional extents
  static const int MAX_RANK = 4;
```

Do you understand the comment here about `num_tuples`?  Either way, we can leave it for our smarter future selves.

Reply:
I don't understand it, sorry :(